app/api/route/schemas/user_schemas.py

```diff
@@ -14,10 +14,6 @@ class RoleSchema(request_schema.OrderedSchema):
         user_models.RoleType, description="The name of the role", by_value=True
     )
 
-    # Output only fields
```

Comment:
decided to remove the created_at/updated_at attributes from the role in the openapi schema, since they aren't really a primary resource